JASP-Desktop/JASP-Desktop.pro

```diff
@@ -178,16 +177,21 @@ win32 {
   copyres.commands +=  $$quote(cmd /c xcopy /S /I /Y $${RESOURCES_PATH} $${RESOURCES_DESTINATION})
 
   $$GENERATE_LANGUAGE_FILES {
+    
     maketranslations.commands += $$quote(echo "Generating language Files") &&
-	maketranslations.commands += $$quote($${QTBIN}lupdate.exe -locations none -extensions $${EXTENSIONS} -recursive $${WINPWD} -ts $${SOURCES_TRANSLATIONS}\jasp.po) &&
-	maketranslations.commands += $$quote($${QTBIN}lupdate.exe -locations none -extensions $${EXTENSIONS} -recursive $${WINPWD} -ts $${SOURCES_TRANSLATIONS}\jasp_$${LANGUAGE_CODE}.po) &&
-
-	#cleanup po files
-	maketranslations.commands += $$quote(\"$${GETTEXT_LOCATION}\msgattrib.exe\" --no-obsolete --no-location $${SOURCES_TRANSLATIONS}\jasp.po -o $${SOURCES_TRANSLATIONS}\jasp.po) &&
-	maketranslations.commands += $$quote(\"$${GETTEXT_LOCATION}\msgattrib.exe\" --no-obsolete --no-location $${SOURCES_TRANSLATIONS}\jasp_$${LANGUAGE_CODE}.po -o $${SOURCES_TRANSLATIONS}\jasp_$${LANGUAGE_CODE}.po)  &&
-
-	#Create jasp_$${LANGUAGE_CODE}.qm
-	maketranslations.commands += $$quote($${QTBIN}lrelease.exe $${SOURCES_TRANSLATIONS}\jasp_$${LANGUAGE_CODE}.po -qm $${RESOURCES_TRANSLATIONS}\jasp_$${LANGUAGE_CODE}.qm) &&
+	maketranslations.commands += $$quote($${QTBIN}lupdate.exe -locations none -extensions $${EXTENSIONS} -recursive $${WINPWD} -ts $${SOURCES_TRANSLATIONS}\jasp.po) &&	#cleanup po files
+    maketranslations.commands += $$quote(\"$${GETTEXT_LOCATION}\msgattrib.exe\" --no-obsolete --no-location $${SOURCES_TRANSLATIONS}\jasp.po -o $${SOURCES_TRANSLATIONS}\jasp.po) &&
+    
+    for(LANGUAGE_CODE, SUPPORTED_LANGUAGES) {
+       message("*********** Busy with languagecode $${LANGUAGE_CODE} ******************")
```

Comment:
This should be something like `$$quote(echo "Generating language Files");` otherwise it is printed when qmake is run and not when you are actually "busy with it" 

https://www.youtube.com/watch?v=oPQ3o14ksaM